conbench/templates/c-benchmarks.html

```diff
@@ -19,12 +22,21 @@ <h4>by name</h4>
       {% endfor %}
     </div>
     <div class="col">
-      <h4>by most recent result (top 15)</h4>
-      {% for benchmark_name, results in benchmarks_by_name_sorted_by_most_recent_result_topN.items() %}
-        <strong><a href="{{ url_for('app.show_benchmark_cases', bname=benchmark_name) }}">{{ benchmark_name }}</a></strong>
-        ({{ results|length }} results)
+      <h4>by most recent result</h4>
+      {% for result in newest_result_for_each_benchmark_name_topN %}
+        <strong><a href="{{ url_for('app.show_benchmark_cases', bname=result.benchmark_name) }}">{{ result.benchmark_name }}</a></strong>
+        (<time class="timeago" datetime="{{ result.started_at_iso }}">{{ result.ui_time_started_at }}</time>)
         <br>
       {% endfor %}
     </div>
   </div>
 {% endblock %}
+{% block scripts %}
+  {{ super() }}
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery-timeago/1.6.7/jquery.timeago.min.js"></script>
+  <script type="text/javascript">
+  $(document).ready(function() {
+     $("time.timeago").timeago();
+  });
```

Comment:
This is https://github.com/rmm5t/jquery-timeago / http://timeago.yarp.com/. I think it has good tooltip handling and timezone awareness. 

I looked at and chose (for now) among
- https://stackoverflow.com/a/69122877/145400
- https://github.com/hustcc/timeago.js
- https://github.com/pragmaticly/smart-time-ago
- https://github.com/mattbradley/livestampjs

This CDN is good: https://cdnjs.com/libraries/jquery-timeago